src/components/dynamic/user/Team.jsx

```diff
@@ -37,6 +37,10 @@ const Team = ({ user, setUser }) => {
   };
 
   const handleJoin = () => {
+    if (id.team == null) {
```

Comment:
triple equals and it should just be checking for empty string and not null

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/93837219/75373691-4f68-4282-9aed-990133ed8aea)
That doesn't seem to work. Instead, this jumps to the very last else statement on line 55.
![image](https://github.com/acm-ucr/hackathon-website/assets/93837219/8d68f68a-638c-41e2-ab0a-8605110cab9f)

Reply:
On the other hand, !id.team seems to work, did you prefer that implementation instead?

Reply:
what are the initial values for the id.team?

Reply:
it's probably empty

Reply:
check the values and do a type check for it instead of a general null check